app/helpers/safe-markup.js

```diff
@@ -1,6 +1,12 @@
 import Ember from 'ember';
 
-export function safeMarkup(params/*, hash*/) {
+/**
+  * safeMarkup helper. Returns a string that will not be HTML escaped by Handlebars.
```

Comment:
A good start! Can you please add a sentence that discourages the use of this helper for user-entered text?

This was created for a very specific niche need (branding), and for security reasons should almost never be used in general development. This came up recently or I wouldn't impose on you.